### DISCUSSION TITLE
Querying profiles as non-admin

### DISCUSSION THREAD

**Discussion by NilsChr** [pocketbase/pocketbase#563] [Q&A] [answered]
I have a data model ad want to add profile id's to a list then use this to check if if a user has access to said resource. 

I have managed to create all the necessary api rules and I'm now working on the functionality for adding a user through my UI. 
My first intention was to add users by their profile username but it seems that it's not possible to get profiles without being admin:

ex:
`const record = client.records.getFullList('profiles')`
returns 
`{
    "code": 403,
    "message": "Only admins can perform this action.",
    "data": {}
}`

What is the best way to tackle this? 

Imagine my record being like this: 
{
     ...
     viewers: <profile_ids>[]
}

**Comment by ganigeorgiev** [comment, marked as answer]
The `profiles` collection has its own API rules. You can edit them by going to:
`Users > Edit profiles collection (settings cogwheel) > API rules` and enable the "List" action rule.

**Reply by ganigeorgiev** [reply]
_Please note that this will change and will be simplified in the near future and the user models and `profiles` collection will be merged per #376._

**Reply by NilsChr** [reply]
thanks for the super quick reply. 
I made it work, thanks :)
